server/controllers/postController.ts

```diff
@@ -10,8 +10,8 @@ async function postPosts(req, res, next) {
 	const username = await userManager.findRefreshToken(refreshToken);
 	const { body: { title, body: postBody, parent, date } } = req;
 	if (date) {
-		const jobSuccess = await scheduleJob({ date, title, body: postBody, author: username, parent }, function(postManager, argsObj: Partial<IPost>) {
-			return postManager.addPost(argsObj.title, argsObj.body, argsObj.author, argsObj.date, argsObj.parent);
+		const jobSuccess = await scheduleJob({ date, title, body: postBody, author: username, parent }, function(postManager, userManager, argsObj: Partial<IPost>) {
```

Comment:
You don't need to make duplication, but rather be consistent.

```suggestion
		const jobSuccess = await scheduleJob({ date, title, body: postBody, author: username, parent }, function(postManager, userManager, argsObj) {
```